lib/src/map/options/enabled_gestures.dart

```diff
@@ -0,0 +1,254 @@
+import 'package:flutter_map/flutter_map.dart';
+import 'package:meta/meta.dart';
+
+@immutable
+class EnabledGestures {
+  /// Private constructor, use the constructors [InteractiveFlags.all] or
+  /// [InteractiveFlags.none] instead to enable or disable all gestures by default.
+  /// If you want to define your enabled gestures using bitfield operations,
+  /// use [InteractiveFlags.bitfield] instead.
+  const EnabledGestures._({
```

Comment:
Not sure why this constructor should be kept private 🤔

Reply:
We can make this constructor public too. 👍🏼 Currently users need to choose if they want to use a whitelist or blacklist approach on allowed gestures but that's not really necessary.

Reply:
Made the default constructor public.